src/OVAL/probes/SEAP/sexp-datatype.c

```diff
@@ -110,12 +110,16 @@ SEXP_datatypePtr_t *SEXP_datatype_get (SEXP_datatypeTbl_t *t, const char *k)
                 if (d->dt_flg & SEXP_DTFLG_LOCALDATA) {
                         struct SEXP_datatype_extptr *eptr = NULL;
 
+#ifdef HAVE_POSIX_MEMALIGN
```

Comment:
Would it be possible to declare a `get_possibly_aligned_memory` with platform-dependent implementation and call that one from now on, so there are no further `#ifdef`s in the code?

Reply:
+100, make it inlined too please

Reply:
Maybe it would be even better to get rid of memory hacks.

Reply:
it's not a memory hack